docs/v5/configuration/htcondor-routes.md

```diff
@@ -113,6 +113,38 @@ To do this, replace the above `SET Requirements` or `set_Requirements` lines wit
     set_Requirements = original_requirements && ...;
     ```
 
+
+### Setting the accounting group according to the the credential that submitted the job ###
+
+A common need in the CE is to want to set the accounting identity of the routed job using information from the credential
+of the submitter of the job.  This originally was done using information from the x509 certificate, in particular `X509UserProxyVOName`
+and `x509UserProxySubject`.  With the switch to SCITOKENs, the equivalent job attributes are `AuthTokenIssuer` and `AuthTokenSubject`. 
+
+It is important to understand that the *condor_schedd* treats `AuthTokenSubject` and `AuthTokenIssuer` as secure attributes. The values
+of these attributes cannot be supplied by the *condor_job_router* directly, they will be set based on what credential the *condor_job_router*
+uses to submit the routed job.  Because of this the value of these attributes in the routed job is almost never the same as the value in the
+original job.  This is different from the way the `x509*` job attributes behaved.
+
+Because of this, the default CE config will copy all attributes that match `AuthToken*` to `orig_AuthToken*` before the route transforms are applied.
+
+Example of setting the accounting group from AuthToken or x509 attributes.
+
+=== "ClassAd Transform"
+
+    ```
+    JOB_ROUTER_CLASSAD_USER_MAP_NAMES = $(JOB_ROUTER_CLASSAD_USER_MAP_NAMES) AcctGroupMap
+    CLASSAD_USER_MAPFILE_AcctGroupMap = <path-to-mapfile>
+
+    JOB_ROUTER_TRANSFORM_SetAcctGroup @=end
+       REQUIREMENTS (orig_AuthTokenSubject ?: x509UserProxySubject) isnt undefined
+       EVALSET AcctGroup UserMap("AcctGroupMap", orig_AuthTokenSubject ?: x509UserProxySubject, AcctGroup)
+       EVALSET AccountingGroup join(".", AcctGroup, Owner)
+    @end
+
+    JOB_ROUTER_PRE_ROUTE_TRANSFORMS = $(JOB_ROUTER_PRE_ROUTE_TRANSFORMS) SetAcctGroup
+    ```
+
```

Comment:
Since this example requires a user map file, it'd be helpful to link to the relevant section of the HTCondor manual